src/core/formatters.ts

```diff
@@ -4,6 +4,15 @@ module Plottable {
 
 export type Formatter = (d: any) => string;
 
+/**
+ * This field is deprecated and will be removed in v2.0.0.
+ * 
+ * The number of milliseconds between midnight one day and the next is 
+ * not a fixed quantity.
+ * 
+ * use Date#setDate(Date.getDate() + n) instead.
```

Comment:
Shouldn't this be a dot instead of a "#"? As in: "`Use date.setDate(date.getDate() + numberOfDays) instead.`".